### PR TITLE
Revised footer

### PR DESCRIPTION
**Pull request by hrvalli**
Hello. We've revised the footer for Duke Extend. If you wouldn't mind making sure there's nothing unexpectedly wrong with it and incorporating it, please, that would be great. 

Thanks, and have a nice weekend!

Heather Valli

### CODE REVIEW

File: lms/templates/footer.html [hunk 1]
```diff
@@ -22,18 +22,18 @@
         <div class="link-info">
           <a href="/about" target="_blank">About</a>
         </div>
+         <a href="https://duke.qualtrics.com/jfe/form/SV_cT3srH17yZ7yrqt" target="_blank">Contact Us</a>
+        </div>
```
Comment (bradenmacdonald) [inline lms/templates/footer.html]: There is a `</div>` here, but I think it is missing the corresponding `<div class="link-info">` first. It should look like:

```html
        <div class="link-info">
          <a href="https://duke.qualtrics.com/jfe/form/SV_cT3srH17yZ7yrqt" target="_blank">Contact Us</a>
        </div>
```

Comment (hrvalli) [reply lms/templates/footer.html]: OK. Thank you for catching that.

**Comment by pomegranited** [comment]
Thanks for this @hrvalli !  I tested it on my ginkgo devstack, and am deploying it to your staging server for final review.

**Comment by hrvalli** [comment]
Thank you, Jill!

**Pull request merged**